### PR TITLE
Create contact on signup

### PR DESCRIPTION
**Pull request by mumoc**
Adds callbacks to create postal address and contacts at brightpearl when signin up

### CODE REVIEW

**Comment by mumoc** [comment]
Before merging this, we need to merge https://github.com/stevenbarragan/nacre-old/pull/3

File: app/models/spree/bp_contact.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      @user = user
+      @bp_address_id = bp_address_id
+    end
+
+    def save
+      Nacre::API::Contact.create bp_fields
+      user.brightpearl_id = response[:id]
+    end
+
+
```
Comment (ingilniero) [inline app/models/spree/bp_contact.rb]: Remove this newline.

**Comment by ingilniero** [comment]
LGTM.

**Comment by rodacato** [comment]
LGTM

**Comment by juankiz** [comment]
Please check Travis before you merge this PR.

**Comment by mumoc** [comment]
Its failing due to missing PR at nacre old

**Pull request merged**